robots-module/src/main/java/orwell/tank/EnumConnectionState.java

```diff
@@ -0,0 +1,10 @@
+package orwell.tank;
+
+/**
+ * Created by Michaël Ludmann on 6/15/15.
+ */
+public enum EnumConnectionState {
+    NOT_CONNECTED,
```

Comment:
I am wondering if there could be some better names since CONNECTION_FAILED also means that the it is NOT_CONNECTED. Maybe CONNECTION_NOT_ATTEMPTED could be better ?